.github/workflows/integration_tests.yml

```diff
@@ -7,44 +7,14 @@ on:
       - main
 
 jobs:
-  compile-contracts:
-    name: Compile contracts using optimized rust compiler
-    runs-on: ubuntu-latest
-    container:
-      image: cosmwasm/workspace-optimizer:0.12.6
-    steps:
-      - name: Checkout sources
-        uses: actions/checkout@v3
-      - name: Install tar for cache upload
-        run: apk add --no-cache tar
-      - name: Rust Dependencies Cache
-        uses: actions/cache@v3
-        with:
-          path: |
-            ~/.cargo/bin/
-            ~/.cargo/registry/index/
-            ~/.cargo/registry/cache/
-            ~/.cargo/git/db/
-            target/
-          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
-      - name: Optimize
-        run: optimize_workspace.sh
-      - name: Upload contracts
-        uses: actions/upload-artifact@v3
-        with:
-          name: dao-dao-contract-artifacts-${{ github.event.number }}
-          path: artifacts
-          retention-days: 1
```

Comment:
I removed this job because I dont want to have to recompile the workspace when we run the integration tests, and making it one job makes that easier. Also, we have to wait for the contracts to optimize either way, so it might as well just be one job.

Reply:
Also, while it's nice to have wasm builds uploaded, I realized that it eventually costs money (as opposed to the cache).

Reply:
Actually, nm it looks like storage is free for public repos. Well we still don't need this job.